### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by lbeckman314**
Only uses gen3-certs.yaml for local deployments

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,9 +32,9 @@ change-context:
 check-secrets:
 	@$(eval ACTUAL=$(shell [ -z $(shell readlink Secrets) ] && echo "<empty>" || echo $(shell readlink Secrets)))
 	@[ "$(ACTUAL)" == "Secrets-$(DEPLOY)" ] || \
-	(printf "\033[1mUnexpected Secrets link\033[0m\n"; \
-	 printf "\033[92mExpected Secrets:\033[0m Secrets-$(DEPLOY)\n"; \
-	 printf "\033[93mActual Secrets:\033[0m   $(ACTUAL)\n"; \
```
Comment (quinnwai) [inline Makefile]: Should we keep this formatting? I like the bold and the colors :)

File: Makefile [hunk 2]
```diff
@@ -48,9 +47,9 @@ check-secrets:
 check-context:
 	@$(eval ACTUAL=$(shell kubectl config current-context))
 	@[ $(ACTUAL) == $(CONTEXT) ] || \
-		(printf "\033[1mUnexpected context\033[0m\n"; \
-		 printf "\033[92mExpected context:\033[0m $(CONTEXT)\n"; \
-		 printf "\033[93mActual context:\033[0m   $(ACTUAL)\n"; \
+		(printf "Unexpected context\n"; \
+		 printf "Expected context: $(CONTEXT)\n"; \
+		 printf "Actual context:   $(ACTUAL)\n"; \
```
Comment (quinnwai) [inline Makefile]: same here

File: Makefile [hunk 3]
```diff
@@ -100,28 +98,26 @@ deploy: check-context check-secrets
...
+		-f Secrets/values.yaml \
+		-f Secrets/user.yaml \
+		-f Secrets/fence-config.yaml; \
+	fi
 
 # https://gist.github.com/prwhite/8168133
 help:	## Show this help message
 	@grep -hE '^[A-Za-z0-9_ \-]*?:.*##.*$$' $(MAKEFILE_LIST) | \
-		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m\033[1m%-20s\033[0m %s\n", $$1, $$2}'
+		awk 'BEGIN {FS = ":.*?## "}; {printf "%-20s %s\n", $$1, $$2}'
```
Comment (quinnwai) [inline Makefile]: ^

File: Makefile [hunk 4]
```diff
@@ -100,28 +98,26 @@ deploy: check-context check-secrets
...
-	@zip Secrets-$(TIMESTAMP).zip Secrets
-	@cp Secrets-$(TIMESTAMP).zip $(HOME)/OneDrive/ACED-deployments
-
+		-f Secrets/TLS/gen3-certs.yaml; \
+	else \
+		helm upgrade --install $(DEPLOY) ./helm/gen3 \
+		-f Secrets/values.yaml \
+		-f Secrets/user.yaml \
+		-f Secrets/fence-config.yaml; \
+	fi
```
Comment (quinnwai) [inline Makefile:112]: nice

**Review by quinnwai — CHANGES_REQUESTED** [review]
Just formatting changes that would be nice to keep methinks

File: Makefile (hunk 1 above)
Comment (lbeckman314) [reply Makefile]: 10-4 — reverting to keep formatting...

**Review by quinnwai — APPROVED** [review]

**Pull request merged**